src/os/bluestore/bluestore_tool.cc

```diff
@@ -305,6 +305,9 @@ int main(int argc, char **argv)
   po_positional.add_options()
     ("command", po::value<string>(&action),
         "fsck, "
+        "qfsck, "
+        "allocmap, "
```

Comment:
I do not see rationale for qfsck and allocmap options.
Those were useful for debugs, but now no longer serve purpose.

We transition rocksdb->ncb during initialization in OSD. If any check for reliability of transition is to be performed, it should be done there.

Reply:
I prefer to keep those options to allow for testing and verification as code keeps growing and we can;t assume that is the last change in this area